firefoxThemes/options/options.html

```diff
@@ -8,6 +8,9 @@
 </head>
 <body>
 <h1>Settings</h1>
+<div class="info-container">
+  <a href="https://github.com/doki-theme/doki-theme-web#feature-preview">Show Documentation</a>
```

Comment:
I chose to link to the `Feature Preview` anchor because, when I choose a lower anchor, all of the images load and the page no-longer is at the specified anchor (real annoying!)

Reply:
I'm on the fence about putting documentation into a tooltip that only shows up when hovering over the option. There isn't anything indicating to me that I should hover over something to get more information, other than "Oh I want to try this feature".  But that is me.

> and add a grand ToC somewhere at the beginning of the README file.

Yeah, that sounds like a good idea.